packages/auth/src/wallet.ts

```diff
@@ -35,11 +35,18 @@ export let walletSelectorComponents: WalletSelectorComponents  = {
 */
 export const setupWalletSelectorComponents = async (network?, contractAddress?, options?: { additionalWallets?: Array<WalletModuleFactory> }): Promise<WalletSelectorComponents> => {
   
+  console.log('modules')
+
   const selector = await setupWalletSelector({
     network: network,
     debug: mbjs.keys.debugMode,
     modules: [
-      ...options?.additionalWallets || [],
+      setupMBWallet({
+        networkId: network,
+        relayerUrl: '/api/relay',
+        signInContractId: MINTBASE_CONTRACTS[network],
+        walletUrl: network ==='testnet' ? 'https://testnet.wallet.mintbase.xyz' : 'https://testnet.wallet.mintbase.xyz',
```

Comment:
It looks to me like the wallet url is testnet regardless of the network value? Shouldn't this be more like:

```suggestion
        walletUrl: network === 'testnet' ? 'https://testnet.wallet.mintbase.xyz' : 'SomethingDifferent',
```